templates/zerver/help/configure-message-editing-and-deletion.md

```diff
@@ -49,31 +50,33 @@ settings.
   in [exports](/help/export-your-organization). Deletion
   permanently (and irretrievably) removes the message from Zulip.
 
-You can access the message editing and deletion settings as follows.
-
-### Configure message editing and deletion
+## Configure message editing and deletion
 
 {start_tabs}
 
 {settings_tab|organization-permissions}
 
-4. Under **Message editing**, configure **Allow message editing**,
-   **Who can edit topic of any message**, and **Who can delete their own messages**.
+1. Under **Message editing**, configure:
+
+    - **Allow message editing**
+    - **Who can edit topic of any message**
+    - **Who can delete their own messages**
+    - **Time limit for deleting messages**
 
 {!save-changes.md!}
 
 {end_tabs}
 
-### Configure who can move topics between streams
+## Configure who can move topics between streams
 
 By default, only organization administrators can move topics between
-streams.  You can configure which roles have permission to do so:
+streams. You can configure which roles have permission to do so.
```

Comment:
The defaults vary by organization type; see https://zulip.com/help/setting-up-zulip-for-a-class#roles-and-permissions.

So let's change this part to not mention the default:

> You can configure which roles have permission to move topics between streams.

Reply:
In general, if you see a note about default organization permissions, we should probably just delete it unless it seems really important. In the future, we might customize even more by org type (it's just education vs. everything else so far).